console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx

```diff
@@ -218,93 +255,28 @@ export const ProjectListContainer: React.FC<IProjectListContainerProps> = ({
     console.log("disable the project", project);
   };
 
-  const projectList: IProject[] = [
-    {
-      projectType: ProjectTypes.MESSAGING,
-      name: "namespace_test1.new_space1",
-      displayName: "new_space",
-      type: "standard",
-      namespace: "namespace_test1",
-      plan: "standard-medium",
-      status: StatusTypes.FAILED,
-      authService: "none",
-      isReady: true,
-      creationTimestamp: "2020-01-20T11:44:28.607Z",
-      errorMessageRate: 3,
-      addressCount: 15,
-      connectionCount: 3
-    },
-    {
-      projectType: ProjectTypes.IOT,
-      isEnabled: true,
-      name: "devops_jbosstest1.k8s_iot1",
-      displayName: "k8s_iot",
-      namespace: "devops_jbosstest1",
-      status: StatusTypes.ACTIVE,
-      creationTimestamp: "2020-05-21T14:40:28.607Z",
-      errorMessageRate: 25,
-      deviceCount: 10500,
-      activeCount: 7100
-    },
-    {
-      projectType: ProjectTypes.MESSAGING,
-      name: "namespace_test1.new_space2",
-      displayName: "new_space",
-      type: "brokered",
-      namespace: "namespace_test1",
-      authService: "none",
-      isReady: true,
-      plan: "brokered-small",
-      status: StatusTypes.PENDING,
-      creationTimestamp: "2020-01-20T05:44:28.607Z",
-      addressCount: 27,
-      connectionCount: 3
-    },
-    {
-      projectType: ProjectTypes.IOT,
-      isEnabled: false,
-      name: "namespace_test1.k8s_iot2",
-      displayName: "k8s_iot",
-      namespace: "namespace_test1",
-      status: StatusTypes.ACTIVE,
-      creationTimestamp: "2020-01-20T05:44:28.607Z",
-      addressCount: 27,
-      connectionCount: 3,
-      errorMessages: ["error message", "pod is not ready."]
-    },
-    {
-      projectType: ProjectTypes.MESSAGING,
-      name: "namespace_test1.new_space3",
-      displayName: "new_space",
-      type: "brokered",
-      namespace: "namespace_test1",
-      plan: "brokered-medium",
-      authService: "none",
-      isReady: true,
-      status: StatusTypes.FAILED,
-      creationTimestamp: "2020-05-21T08:44:28.607Z",
-      addressCount: 27,
-      connectionCount: 3,
-      errorMessageRate: 98,
-      errorMessages: ["error message", "issue with operator"]
-    },
-    {
-      projectType: ProjectTypes.MESSAGING,
-      name: "namespace_test1.new_space4",
-      displayName: "new_space",
-      namespace: "namespace_test1",
-      plan: "standard-small",
-      type: "standard",
-      authService: "none",
-      isReady: true,
-      status: StatusTypes.CONFIGURING,
-      creationTimestamp: "2020-05-21T08:44:28.607Z",
-      addressCount: 27,
-      connectionCount: 3,
-      errorMessageRate: 98,
-      errorMessages: ["error message", "issue with operator"]
-    }
-  ];
+  const getProjects = () => {
+    return projects?.map((project: any) => {
```

Comment:
project should have a type.

Reply:
This page will refactor based on single object for msg-project and iot-project. As such we don't have final complex data type for project. Let it be `any`  for time being.